genie-agent/src/test/resources/JobSetupServiceImplSpec_run1.test.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+#
+# Generated by Genie for job: <JOB_ID_PLACEHOLDER>
+#
+
+# Error out if any command fails
+set -o errexit
+# Error out if any command in a pipeline fails
+set -o pipefail
+# Error out if unknown variable is used
+set -o nounset
+
+
+export GENIE_JOB_DIR="<JOB_DIR_PLACEHOLDER>"
+
+export GENIE_APPLICATION_DIR="${GENIE_JOB_DIR}/genie/applications"
```

Comment:
Should we export variables like `GENIE_APPLICATION_0_ID=` `GENIE_APPLICATION_1_ID=` etc?

Reply:
The server is generating and sending these over as part of the specification.
As you can see in the test files we compare against (for example `JobControllerIntegrationTest`, everything is there).